tests/providers/amazon/aws/operators/test_eks.py

```diff
@@ -49,10 +52,41 @@
 NAME_LIST = ["foo", "bar", "baz", "qux"]
 
 
+class clusterParams(TypedDict):
```

Comment:
```suggestion
class ClusterParams(TypedDict):
```

Conventionally types should use CamelCases. (Same for all others below.)

Reply:
Fixed, thanks